src/_data/catalog/widgets.json

```diff
@@ -343,6 +420,17 @@
     "link": "https://api.flutter.dev/flutter/widgets/Column-class.html",
     "image": "<svg viewBox='0 0 100 100'><rect x='0' y='0' width='100' height='100' fill='#3949ab'/><rect x='30' y='10' width='40' height='80' fill='#ffffff'/><rect x='40' y='15' width='20' height='20' fill='#4dd0e1'/><rect x='35' y='40' width='30' height='30' fill='#4dd0e1'/></svg>"
   },
+  {
+    "name": "Common buttons",
+    "description": "Clickable blocks that start an action, such as sending an email, sharing a document, or liking a comment.",
+    "categories": [],
+    "subcategories": [
+      "Actions"
+    ],
+    "link": "https://api.flutter.dev/flutter/material/ButtonStyle-class.html",
```

Comment:
Linking directly to the "Material 3 button types" section could provide a more straightforward result for the user: https://api.flutter.dev/flutter/material/ButtonStyle-class.html#material-3-button-types